_posts/2019-12-17-zero-to-main-rust-1.md

```diff
@@ -0,0 +1,433 @@
+---
+title: "From zero to main(): Bare metal Rust"
+description: "Bringing an embedded Rust application from zero to main"
+author: james
+tags: [zero-to-main]
+---
```

Comment:
Do I need an image for my frontmatter?

Reply:
Image is optional, it helps show a better preview when you link is shared.